t/directorysearch.t

```diff
@@ -72,13 +72,6 @@ $is->(
     "Russia", "loc_cn=RU&opt_sort=ut", LJ::Directory::Constraint::Location->new( country => 'RU' )
 );
 
-$is->(
-    "Age Range + last week",
-"loc_cn=&loc_st=&loc_ci=&ut_days=7&age_min=14&age_max=27&int_like=&fr_user=&fro_user=&opt_format=pics&opt_sort=ut&opt_pagesize=100",
-    LJ::Directory::Constraint::Age->new( from => 14, to => 27 ),
```

Comment:
pst I think you maybe meant to put this change on #3137

Reply:
oh shit you're right